Add guideline on how to contribute to the meetup and extend the contact information (#31)
WebEngDUS/WebEngDUS: CONTRIBUTING.md

# How to contribute to Web Engineering Düsseldorf?

If you seek to be part of the community, contribute something back, and support the local tech community in Düsseldorf, check below how you can support the efforts around Web Engineering Düsseldorf.

## Speaking at the Meetup

We are always looking for speakers to run sessions at our meetups.

> For us, this doesn't matter if you are a newcomer on stage or a highly experienced speaker.

Reviewer: ```suggestion
For us, it doesn't matter if you are a newcomer on stage or a highly experienced speaker.
```

The meetup is there to share knowledge and try things out.

> If you want to
> 
> * gain speaker experience
> * or test one of your conference talks
> * or share your knowledge with the local community
> * or have a topic in mind to speak about
> * or need inspiration on what to talk about at the meetup
> 

Reviewer: ```suggestion
If you

* want to gain experience as a speaker
* want to test one of your conference talks
* like to share your knowledge with the local community
* have a topic in mind to speak about
* need inspiration on what to talk about at the meetup

```

We are happy to help and guide you.

PS: You can get a sneak peek at our [Speaker Briefing](./README.md#speaker-briefing).

## Helping us to organize a Meetup

Organizing a meetup regularly requires effort and time.
At the same time, it is a rewarding experience to provide something back to your local tech community.

If you have ambitions in helping us organize the meetup, get in contact with us (see our [contact information](./README.md#organizers-and-contact), [open a new issue](https://github.com/WebEngDUS/WebEngDUS/issues/new) or speak to us at a local meetup).
We are happy to start the conversation for more support.

## Sponsoring us

The meetup depends on support from (local) companies to make it happen.

### Being a Location-Sponsor

If your company wants to host one of our meetup events, please get in contact with us (see our [contact information](./README.md#organizers-and-contact), [open a new issue](https://github.com/WebEngDUS/WebEngDUS/issues/new) or speak to us at a local meetup).

PS: You can get a sneak peek at our [Location Sponsor / Location and Company Briefing](./README.md#location-sponsor--location-and-company-briefing).

> ### Gifts / Winnings for a raffle

Reviewer: ```suggestion
### Raffle Prices / Speaker Presents
```
